modin/experimental/pandas/__init__.py

```diff
@@ -33,13 +33,21 @@
 """
 
 from modin.pandas import *  # noqa F401, F403
+from modin.utils import func_from_deprecated_location
 
 from .io import (  # noqa F401
     read_csv_glob,
     read_custom_text,
     read_json_glob,
     read_parquet_glob,
-    read_pickle_distributed,
+    read_pickle_glob,
     read_sql,
     read_xml_glob,
 )
+
+read_pickle_distributed = func_from_deprecated_location(
```

Comment:
Could you create an issue regarding the methods we deprecated so we can remove them in a single PR for a future release?

Reply:
https://github.com/modin-project/modin/issues/6967